src/app/cli/src/coda_run.ml

```diff
@@ -83,6 +86,113 @@ let log_shutdown ~conf_dir ~top_logger coda_ref =
         Logger.debug logger ~module_:__MODULE__ ~location:__LOC__ "%s"
           (Visualization_message.bootstrap "transition frontier") )
 
+let make_report_and_log_shutdown exn_str ~conf_dir ~top_logger coda_ref =
+  Logger.info top_logger ~module_:__MODULE__ ~location:__LOC__
+    "Generating crash report" ;
+  let temp_config = Core.Unix.mkdtemp (conf_dir ^/ "coda_config_temp") in
```

Comment:
This looks like an incorrect usage of mkdtemp. http://man7.org/linux/man-pages/man3/mkdtemp.3.html

Reply:
I was a lil confused by that as well. The tmp folder was created in my src directory by default. So added the absolute path. Could it be a mac thing? I guess I could leave out the path since I am deleting the directory at the end.

Reply:
I'm pretty sure that when we try to run this on linux, the syscall will emit an error when there isn't a "XXXXXX" at the end of the template string (according to the man page), so I think you should at minimum add that.

Reply:
Cool, I'll add that for now. Also, https://ocaml.janestreet.com/ocaml-core/latest/doc/core/Core__/Core_unix/index.html#val-mkdtemp doesn't mention the template thing